src/translations/sqlb_de.ts

```diff
@@ -3029,7 +3029,7 @@ Sie können SQL-Statements aus einer Objektzeile fassen und in anderen Anwendung
     <message>
         <location filename="../MainWindow.ui" line="1879"/>
         <source>Ctrl+Shift+O</source>
-        <translation></translation>
+        <translation>Strg+Umschalt+0</translation>
```

Comment:
```suggestion
        <translation></translation>
```

Keyboard shortcuts get automatically localized by Qt, and translating key names may indeed break them. They show up on the translation file only for the rare occasions that one has to redirect a shortcut to another one in a specific language. Further information: https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts